src/torchcodec/encoders/_video_encoder.py

```diff
@@ -110,6 +137,7 @@ def to_file_like(
         pixel_format: Optional[str] = None,
         crf: Optional[Union[int, float]] = None,
         preset: Optional[Union[str, int]] = None,
+        codec_options: Optional[Dict[str, str]] = None,
```

Comment:
Could we make it `Dict[str, Any]`? I think there's value in allowing `"some_param":16` instead of the un-pythonic `"some_param: "16"`. We just need to call `str()` on all the values, which is OK.